### DISCUSSION TITLE
⛔ Show the full image when opened with the Lightbox

### DISCUSSION THREAD

**Discussion by nicopra** [elementor/elementor#18127] [Feature Request]
It seems that the lightbox zoom works by scaling the image using CSS animations.
Would be great that it could display the image at full resolution without scaling so it doesn't appears blurry when zoomed.

**Comment by nicholaszein** [comment]
Hello.

The Elementor Lightbox always shows the full image size when you open it, regardless of the size you've chosen in the Image widget.

Make sure your images have enough resolution when uploading them. A good size would have at least 1080px of height.

Kind regards
